database/README.md

```diff
@@ -0,0 +1,53 @@
+# Database
+This bot persists data in a single SQLite database (which is a file in this directory).
+
+### Keeping Your Local Database Up-to-Date
+- [`schema.py`](./schema.py) is responsible for creating the database & its tables.
+It will not overwrite any existing tables.
+This file is executed every time that the bot runs, so you don't need to worry about running it manually.
+- [`migrate.py`](./migrate.py) is responsible for updating an old database to a new format. It should run **once** after
+a schema change occurs via the terminal. Be careful not to miss a schema update, or you may need to delete your database
+file and let [`schema.py`](./schema.py) create it again from scratch (losing all your data).
+
+
+### Making a Change to the Production Database
+- Update [`schema.py`](./schema.py) so that it can create a database in the correct state, from scratch.
+- Overwrite [`migrate.py`](./migrate.py) such that it applies your change to the previous version of your database.
+
```

Comment:
If contributing these changes officially to pax, mention that ERD.mdj must be updated as well

Reply:
I added a note in e399501b2bf029864bb3494b823fe40bb522d3be. Is it okay to say this?
> Reach out to @skagame on Discord if you need help with this.

I'd understand if you didn't want to be solely responsible for this file, I just don't know how to generate it, haha.

Reply:
Yup, lgtm